src/main/java/com/ciandt/techgallery/service/GooglePlusCommunicationService.java

```diff
@@ -0,0 +1,53 @@
+package com.ciandt.techgallery.service;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+
+import com.ciandt.techgallery.service.enums.FeatureEnum;
+import com.google.api.server.spi.response.BadRequestException;
+import com.google.api.server.spi.response.InternalServerErrorException;
+import com.google.api.server.spi.response.NotFoundException;
+import com.google.appengine.api.users.User;
+
+/**
+ * Services for google plus communications.
+ * 
+ * @author Thulio Ribeiro
+ *
+ */
+public interface GooglePlusCommunicationService {
```

Comment:
@tsr-thulio I suggest changing the class name to something more generic. Maybe in the future TechGaller must post on other Social networks.
I think SocialNetworkCommunicationService. The implementation class name is great!